.github/workflows/build-release.yml

```diff
@@ -0,0 +1,66 @@
+name: Build & Release draft
+
+on:
+    pull_request:
+    push:
+        branches:
+            - master
+
+jobs:
+    deploy:
+        name: build dependencies & create artifact
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2.0.0
+            #- name: Build JS dependencies
+            #  uses: PrestaShopCorp/github-action-build-js/12@v1.0
+            #  with:
+            #      cmd: yarn
+            #      path: ./_dev
+            - name: Install composer dependencies
+              run: composer install --no-dev -o
+            - name: Clean-up project
+              uses: PrestaShopCorp/github-action-clean-before-deploy@v1.0
```

Comment:
In order to keep the open source project and the company separated, shall we move/fork https://github.com/prestashopcorp/github-action-clean-before-deploy inside PS OSS org ?

Reply:
Ping @eternoendless & @Quetzacoalt91 : What do you think about this ?

Reply:
Hmm unfortunately, I don't see how this would help us. Either we double the work to keep these repo up to date, or we basically move the issue by loading `PrestaShopCorp/github-action-clean-before-deploy` from PrestaShopCorp repositories.